src/routes/game/components/GameMenu.vue

```diff
@@ -111,12 +116,12 @@ export default {
       },
     },
     dialogTitle() {
-      return this.showConcedeDialog ? 'Concede' : 'Request Stalemate';
+      return this.showConcedeDialog ? this.$t('game.menus.gameMenu.concede') : this.$t('game.menus.gameMenu.stalemate');
     },
     dialogText() {
       return this.showConcedeDialog
-        ? 'The game will end and your opponent will win.'
-        : 'If your opponent agrees, the game will end in a stalemate. The request will cancel if the opponent declines or either player makes a move';
+        ? this.$t('game.menus.gameMenu.concedeDialog')
+        : this.$t('game.menus.gameMenu.stalemateDialog');
```

Comment:
`return this.t(this.showConcedeDialog ? 'game.menus.gameMenu.concedeDialog' : 'game.menus.gameMenu.stalemateDialog') `